src/fiatconnect/ReviewScreen.tsx

```diff
@@ -323,8 +323,8 @@ export default function FiatConnectReviewScreen({ route, navigation }: Props) {
                   serializablePreparedTransaction:
                     prepareTransactionsResult.result?.type === 'possible'
                       ? getSerializablePreparedTransaction(
-                          prepareTransactionsResult.result.transactions[0] // there should be only one transaction
```

Comment:
@crazeteam, you have a lint error here; mind running `yarn lint` in the repository's root and committing the changes.